packages/broadcast/component.js

```diff
@@ -1,46 +1,56 @@
-import { FabricWebComponent, windowExists } from '../utils';
+import { LitElement } from 'lit-element';
+import { windowExists } from '../utils';
+import { FabricToastContainer } from '../..';
 
-export class FabricBroadcast extends FabricWebComponent {
-  async connectedCallback() {
-    const REFETCH_INTERVAL = Number(this.getAttribute('interval')) || 300000; // 300 000 ms = 5 minutes
+export class FabricBroadcast extends LitElement {
+  static properties = {
+    _messages: {
+      state: true,
+      hasChanged(newVal, oldVal) {
+        if (!oldVal) return true;
+        const newIds = newVal.map(({ id }) => id).sort();
+        const oldIds = oldVal.map(({ id }) => id).sort();
+        return JSON.stringify(newIds) !== JSON.stringify(oldIds);
+      },
+    },
+    interval: { type: Number, attribute: true, reflect: true },
+    url: { type: String, attribute: true, reflect: true },
+    api: { type: String, attribute: true, reflect: true },
+  };
 
-    // Fetch broadcast on load and setup refetch
-    await this.fetchMessage();
-    setInterval(async () => await this.fetchMessage(), REFETCH_INTERVAL);
+  constructor() {
+    super();
+    this._messages = [];
+    this.interval = 30000;
+    this.url = windowExists ? window.location.href : '';
+    this._toastContainer = FabricToastContainer.init();
```

Comment:
initialises the FabricToastContainer to utilise its get/set/del methods to adding, updating and removing toasts